github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelAddedEvent/help.html

```diff
@@ -1,4 +1,5 @@
 <div>
-    Trigger when label was added to PR. When multiple label expected,
-    then job triggered only when all labels will appear in summary (in comparison to previous state).
+    Triggers a build when the label or labels specified are added to a
+    pull request. Note that with multiple labels, all of the labels must be
+    present to cause the trigger to fire.
```

Comment:
I tried document all things in javadocs, that's annoying copy-pasting everything to help pages :(